fractions/javaee/jaxrs-jsonb/src/main/resources/modules/org/wildfly/swarm/jaxrs/jsonp/main/module.xml

```diff
@@ -0,0 +1,5 @@
+<module xmlns="urn:jboss:module:1.3" name="org.wildfly.swarm.jaxrs.jsonb">
+  <dependencies>
+    <module name="org.jboss.resteasy.resteasy-json-binding-provider"/>
+  </dependencies>
+</module>
```

Comment:
Should there be `module.conf` instead of this?

Reply:
This is how it is done in jaxrs-jsonp. But I'll try

Reply:
@Ladicek it worked, with module.conf only